README.md

```diff
@@ -1,3 +1,98 @@
-# Triton-optimized Couchbase
+# Autopilot pattern for Couchbase
 
-Read [the blog post](https://www.joyent.com/blog/couchbase-in-docker-containers) and the [Docker Compose yaml](https://github.com/misterbisson/clustered-couchbase-in-containers) to better understand how to use this image.
\ No newline at end of file
+Read [the blog post](https://www.joyent.com/blog/couchbase-in-docker-containers) and the [Docker Compose yaml](https://github.com/misterbisson/clustered-couchbase-in-containers) to better understand how to use this repo.
+
+This is a Dockerfile, Docker Compose file and shell script that will deploy a Couchbase cluster that can be scaled easily using `docker compose scale couchbase=$n`.
+
+## Prep your environment
+
+1. [Get a Joyent account](https://my.joyent.com/landing/signup/) and [add your SSH key](https://docs.joyent.com/public-cloud/getting-started).
+1. Install and configure the [Joyent CloudAPI CLI tools](https://apidocs.joyent.com/cloudapi/#getting-started).
+1. [Install Docker](https://docs.docker.com/installation/#installation) and [Docker Compose](https://docs.docker.com/compose/install/).
+1. [Configure your Docker CLI and Compose for use with Joyent](https://apidocs.joyent.com/docker):
+
+```
+curl -O https://raw.githubusercontent.com/joyent/sdc-docker/master/tools/sdc-docker-setup.sh && chmod +x sdc-docker-setup.sh
+ ./sdc-docker-setup.sh -k us-east-1.api.joyent.com <ACCOUNT> ~/.ssh/<PRIVATE_KEY_FILE>
+```
+
+## Easy instructions
+
+1. [Clone](git@github.com:misterbisson/clustered-couchbase-in-containers.git) or [download](https://github.com/misterbisson/clustered-couchbase-in-containers/archive/master.zip) this repo.
```

Comment:
A bunch of `misterbisson` namespace stuff throughout, but I think you're on that.